server/src/weaverbird/pipeline/pipeline.py

```diff
@@ -198,7 +198,6 @@ def dict(self, *, exclude_none: bool = True, **kwargs) -> dict:
     "prevRank",  # for the rank step, this needs to be kept
     "$ne",  # for isnotnull (None -> null for mongo),
     "$eq",  # for isnull (None -> null for mongo)
-    "pipeline",
```

Comment:
Why has this behaviour been changed ?

Reply:
because the same behaviour with pipeline is the one i was covering here : https://github.com/ToucanToco/weaverbird/blob/42c08d9a704e593fc8ddd4199ea92c432bbfd808/server/src/weaverbird/pipeline/pipeline.py#L311-L314